### PR TITLE
Kafka Producer

### PR DESCRIPTION
**Pull request by dendisuhubdy**
- [x] adding FIX messages to kafka producer

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -155,6 +162,10 @@ class StatsEndpoint {
         new_order.push_back_char      (hffix::tag::TimeInForce, '1');
 
         new_order.push_back_trailer(); // write CheckSum.
+        
+        std::string message(buffeer);
+        producer.produce(MessageBuilder("fix_message_queue").partition(0).payload(message));
```
Comment (dendisuhubdy) [inline src/main.cpp]: @Ujang360 here bro

Comment (Ujang360) [reply src/main.cpp]: Cool! By the way, is `buffeer` a typo?

Comment (dendisuhubdy) [reply src/main.cpp]: Nyeeetttt

File: src/main.cpp [hunk 2]
```diff
@@ -232,6 +243,10 @@ class StatsEndpoint {
         new_order.push_back_char      (hffix::tag::TimeInForce, '1');
 
         new_order.push_back_trailer(); // write CheckSum.
+        
+        std::string message(buffeer);
```
Comment (Ujang360) [inline src/main.cpp]: One more 😸

**Comment by nevrending** [comment]
@dendisuhubdy LGTM
Aku coba bantu fix indentation sama remove whitespaces yah

**Comment by dendisuhubdy** [comment]
Thanks guys nice one

**Pull request merged**